infra/nnfw/python/setup.py

```diff
@@ -0,0 +1,58 @@
+from setuptools import setup, find_packages
+import os
+import shutil
+
+architecture_directory = ['x86_64', 'armv7l', 'aarch64']
+package_directory = 'nnfwapi'
+packaging_directory = ['build', 'dist', package_directory + '.egg-info']
+so_list = []
+
+try:
+    # remove packaging directory
+    for packaging_dir in packaging_directory:
+        if os.path.exists(packaging_dir):
+            print(f"Deleting existing directory '{packaging_dir}'...")
+            shutil.rmtree(packaging_dir)
```

Comment:
Why do we need `for` loop? Why don't you remove `packaging_directory` at once?

Reply:
Except for the directories in "packaging_directory", some files like setup.py and pack.sh were intended not to be deleted..! 
Do you think it's better to write code in a way other than `for` loop?

Reply:
I don't understand the aim of this PR, maybe due to my lack of knowledge about python packaging. I expected this PR is for script to generate whl, but it seems not.

Reply:
I didn't fully understand your comment saying that  "README.md requires to install prebuilt python package".
By using `setup.py` with `python3 setup.py sdist bdist_wheel` command in `pack.sh`, nnfwapi package is created. It makes the so files into a deployable state. After uploading this package using`twine`, anyone can install and use packages through PyPI without having to build themselves.

Reply:
> ## Requirement
>
> `nnfwapi` should be installed from PyPI.
> ```
> $ pip install nnfwapi
> ```

I just followed the guide of `README.md`. It requires to install `nnfwapi`.